bitcoin-rpc/src/main/java/com/msgilligan/bitcoin/rpc/BitcoinClient.java

```diff
@@ -83,7 +84,7 @@ public Boolean waitForServer(int timeout) throws JsonRPCException {
                 status = e.getMessage();
             } catch (JsonRPCStatusException e) {
                 // If server is in "warmup" mode, e.g. validating/parsing the blockchain...
-                if ( e.jsonRPCCode == -28) {
+                if (e.jsonRPCCode == -28) {
```

Comment:
We should probably start defining error codes symbolically, but this is fine for now.

(I realize that I put in the -28 and you just removed a space)